.travis.yml

```diff
@@ -17,6 +17,7 @@ env:
   - REXRAY_BUILD_TYPE=agent
   - REXRAY_BUILD_TYPE=controller
   - REXRAY_BUILD_TYPE= DRIVERS=ebs
+  - REXRAY_BUILD_TYPE= DRIVERS=efs
```

Comment:
Hi @cduchesne,

The fact the rest were in alphabetical order were happy accidents. You just couldn't resist maintaining that though, could you? :)